app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt

```diff
@@ -0,0 +1,38 @@
+package org.fossasia.openevent.general.speakers
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.github.jasminb.jsonapi.LongIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("speaker")
+@Entity
+data class Speaker(
+    @Id(LongIdHandler::class)
+    @PrimaryKey
+    val id: Long,
+    val name: String?,
+    val email: String?,
+    @JsonProperty("photo-url")
+    val photoUrl: String?,
+    @JsonProperty("short-biography")
+    val shortBiography: String?,
+    @JsonProperty("long-biography")
```

Comment:
Same

Reply:
i'm using the short bio and if we want to have sepeare speaker fragment we'll be needing these properties

Reply:
I'm talking about the annotation

Reply:
but without the annotation it won't map as it contains "-" and I can't name as "short-biography"

Reply:
yes i've tried it not working without annotation

Reply:
It should work, as the NamingStrategy is set to SnakeCase

Reply:
i tried with the photo-url and it wasn't able to recognize it

Reply:
@iamareebjamal i got it i need to annotate clas with jsonnaming

Reply:
and it was kebabCase